app/models/had_skill.rb

```diff
@@ -4,7 +4,7 @@ class HadSkill < ActiveRecord::Base
   validates :user_id,
     null: false
   validates :expertise_level,
-    inclusion: { in: ["", "Beginner", "Intermediate", "Advanced"]}
+    inclusion: { in: ["", "Beginner", "Intermediate", "Advanced", nil]}
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Space inside } missing.